images/installer/Dockerfile.upi.ci

```diff
@@ -40,6 +40,10 @@ RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/
 RUN curl -L -O https://github.com/vmware/govmomi/releases/download/v0.20.0/govc_linux_amd64.gz && \
     gzip -d govc_linux_amd64.gz && \
     chmod +x govc_linux_amd64 && mv govc_linux_amd64 /bin/govc
+RUN curl -L -O -k https://vcsa.vmware.devcluster.openshift.com/certs/download.zip && \
```

Comment:
@patrickdillon `vcsa-ci` 
`vcsa` is the dev environment

Reply:
fixed! thank you